Standards/scs-0217-v1-baseline-cluster-security.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Kubernetes cluster baseline security
+type: Standard
+status: Draft
+track: KaaS
+---
+
+## Introduction
+
+A Kubernetes instance is provided as a cluster, which consists of a set of machines,
```

Comment:
I'm not sure this first sentence is needed. I assume the target audience of this standard is people at least superficially familiar with kubernetes? It might be good to instead mention the target audience in the standard, but afaik we do not do this in our standards.

Nevertheless, I guess this sentence is redundant information for the target audience which doesn't really add anything to the standard.

Reply:
I can understand that. I thought it was better for general text flow, but I don't need the sentence in, so I removed it.
But I don't have a "better" introduction, so we will keep with that.